suite-common/wallet-utils/src/accountUtils.ts

```diff
@@ -669,6 +669,26 @@ export const getTokensFiatBalance = (
     return totalBalance.toFixed();
 };
 
+export const getAssetTokensFiatBalance = (
+    accounts: Account[],
+    localCurrency: FiatCurrencyCode,
+    rates?: RatesByKey,
+) => {
+    let totalBalance = new BigNumber(0);
+
+    accounts.forEach(account => {
```

Comment:
would be nicer to use reduce

```
    accounts.reduce((total, account) => {
        const tokensBalance = getAccountTokensFiatBalance(
            account,
            localCurrency,
            rates,
            account.tokens,
        );
        return total.plus(tokensBalance ?? 0);
    }, new BigNumber(0)).toFixed();
```

if it works

Reply:
Yes with a slight change, here in a fixup
https://github.com/trezor/trezor-suite/pull/14996/commits/b2b5096161bd3f365cb8e613cc7f173d8a16d47a